config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1beta1.yaml

```diff
@@ -288,7 +288,6 @@ presubmits:
       testgrid-tab-name: capz-pr-conformance-v1beta1
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
   - name: pull-cluster-api-provider-azure-apidiff-v1beta1
-    cluster: eks-prow-build-cluster
```

Comment:
I think this one can stay (with the increased CPU) since it doesn't reference the `prowjob-default-sa` service account.

Reply:
Or we can revert it to keep both apidiff tests the same. Your call.

Reply:
ACK

I wonder if the service account reference above is actually necessary (these apidiff presubmit jobs are doing the same thing, just on different branches, if one branch isn't referencing the service count and is running as intended I suspect the reference is cruft)

Stand by...